wpeverest/everest-forms
# Fix - Selective html tag support in field label

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://wordpress.org/support/topic/create-hyperlink-in-title/

### How to test the changes in this Pull Request:

1. Create a default contact form.
2. Modify the label with hyperlinks, i.e, 
`<a href="#" class="link"><span class="tooltip"><em><strong>Help</strong></em></span></a>`
3. Save and preview, now label should should support links.

Note: This feature is for only selective developers who want to integrate tooltip, lightbox links independently.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Selective html tag support in field label.
